swagger-api/src/main/java/org/xwiki/contrib/swagger/script/SwaggerScriptService.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.contrib.swagger.script;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.xwiki.component.annotation.Component;
+import org.xwiki.contrib.swagger.internal.Proxy;
+import org.xwiki.script.service.ScriptService;
+import org.xwiki.stability.Unstable;
+
+/**
+ * Simple scripting service for accessing the proxy.
+ *
+ * @version $Id$
+ * @since 1.0
+ */
+@Component
+@Named("swagger")
+@Singleton
+@Unstable
```

Comment:
Honestly, unless you have a strict policy regarding unstable API and know where to remove them (like we have in XWiki Standard), I'm not sure using the `@Unstable` annotation in contrib extension makes that much sense (the problem is that there is a good chance it stay there forever).